src/app/CommandHandler.h

```diff
@@ -59,6 +59,53 @@ class CommandHandler : public Command
         virtual void OnDone(CommandHandler * apCommandObj) = 0;
     };
 
+    class Handle
```

Comment:
Can we use `ReferenceCountedHandle` in `src/lib/support/ReferenceCountedHandle.h` here ?